.devcontainer/setup.sh

```diff
@@ -12,6 +12,14 @@ if [[ ! -d "vendor" ]]; then
 fi
 occ app:enable libresign
 if [[ ! -d "node_modules" ]]; then
+	occ theming:config name "LibreSign"
+	occ theming:config url "https://libresign.coop"
+	occ theming:config primary_color "#144042"
+	occ config:app:set libresign extra_settings --value=1
+	occ config:system:set defaultapp --value libresign
+	occ config:system:set has_valid_subscription --type=boolean --value=true
+	occ config:system:set knowledgebaseenabled --value false --type boolean
```

Comment:
Thinking more, could be good remove this. Is a very specific scenario that haven't a purpose at development environment because don't do any effect to developers.
```suggestion
```

Reply:
Removed.